src/puppetlabs/trapperkeeper/services/status/status_service.clj

```diff
@@ -40,10 +40,11 @@
       (add-ring-handler this handler))
     (let [debug-logging-config (schema/validate status-core/DebugLoggingConfig
                                                 (get-in-config [:status :debug-logging]))
-          debug-logging-interval (:interval-milliseconds debug-logging-config)]
-      (when debug-logging-interval
-        (log/info "Starting background logging of status data")
-        (interspaced debug-logging-interval status-logging/log-status)))
+          interval-minutes (:interval-minutes debug-logging-config)]
+      (when interval-minutes
+        (let [interval-milliseconds (* 60000 interval-minutes)]
```

Comment:
This is totally fine like this, but just FYI, Java's built-in `TimeUnit` is pretty nifty for stuff like this:

```clj
(.toMillis TimeUnit/MINUTES 30)
=> 1800000
```

Just something good to know about.